agents/ocf/HealthCPU.in

```diff
@@ -67,6 +67,15 @@ the #health-cpu will go red if the %idle of the CPU falls below 10%.
 <content type="string" default="10"/>
 </parameter>
 
+<parameter name="attrd_delay" reloadable="1">
```

Comment:
I'm thinking "dampening" or "delay" for a more intuitive name

Reply:
It does seem more appropriate，I've changed it to “dampening”.